lib/OpenQA/Utils.pm

```diff
@@ -90,6 +90,7 @@ our @EXPORT = qw(
   hashwalker
   read_test_modules
   feature_scaling
+  label
```

Comment:
This name is to generic to export it by a module which we import almost everywhere.

Reply:
job_label?